lib/datetimeWrapper.js

```diff
@@ -317,9 +307,82 @@ var datetimeWrapper = function(moment) {
     /*
      * Get all timezone objects
      *
-     * @return {Array} a list of available timezone objects
+     * @return {Object} a hash of arrays of timezone objects
+     * in convenient categories for displaying in a searchable dropdown
      */
     getTimezones: function() {
+      var tzNames = moment.tz.names();
+      var timezones = {
+        // main timezones of the U.S. lower 48
+        bigFour: [],
+        // remainder of U.S. timezones
+        unitedStates: [],
+        // the English-speaking world + Europe
+        hoisted: [],
+        // everywhere else
+        theRest: []
+      };
+      var big4 = {
+        'US/Eastern': true,
+        'US/Central': true,
+        'US/Mountain': true,
+        'US/Pacific': true
+      };
+      var newZealand = {
+        'Pacific/Auckland': true,
+        'Pacific/Chatham': true
+      };
+      var now = new Date().valueOf();
+      for (var i = 0; i < tzNames.length; ++i) {
+        var zone = moment.tz.zone(tzNames[i]);
+        // we use the current epoch time to determine the current UTC
+        // offset for each timezone (i.e., adjust for DST or not)
+        // the offset is part of the `label` attribute
+        var offsetInHours = moment.duration(zone.offset(now), 'minutes')
+          .asMilliseconds()/(1000*60*60);
+
+        // because moment signs offsets the opposite way from the ISO 8601 standard
+        // the standard is negative for N. American timezones
+        // but moment (and JavaScript in general) represent them as positive
+        var offsetAsString = offsetInHours > 0 ?
+          '-' + offsetInHours : '+' + Math.abs(offsetInHours);
+
+        var timezone = {
+          label: format('%s (UTC%s)', tzNames[i], offsetAsString),
+          value: tzNames[i],
+          offset: Number(offsetAsString)
+        };
+
+        if (big4[tzNames[i]] === true) {
+          timezones.bigFour.push(timezone);
+        }
+        else if (tzNames[i].search('US/') !== -1) {
+          timezones.unitedStates.push(timezone);
+        }
+        else if (tzNames[i].search('Canada') !== -1) {
+          timezones.hoisted.push(timezone);
+        }
+        else if (tzNames[i].search('Europe') !== -1) {
+          timezones.hoisted.push(timezone);
+        }
+        else if (tzNames[i].search('Australia') !== -1) {
+          timezones.hoisted.push(timezone);
+        }
+        else if (newZealand[tzNames[i]] === true) {
+          timezones.hoisted.push(timezone);
+        }
+        // reformat these a tiny bit so that 'New Zealand' is searchable
+        else if (tzNames[i].search('NZ') !== -1) {
+          timezone.label = timezone.label.replace('NZ', 'New Zealand');
+          timezones.hoisted.push(timezone);
+        }
+        // there's a whole bunch of timezones listed as "Etc/GMT+1"
+        // that clutter things up a lot...
+        else if (tzNames[i].search('GMT') !== -1) {}
```

Comment:
it looks like there are still a bunch of weird ones in there like: `PST8PDT`, `EST`.  Not sure if we want to filter these or not.